core/src/com/dragons/game/model/Object.java

```diff
@@ -0,0 +1,88 @@
+package com.dragons.game.model;
+
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.physics.box2d.CircleShape;
+import com.badlogic.gdx.physics.box2d.PolygonShape;
+import com.badlogic.gdx.physics.box2d.Shape;
+import com.badlogic.gdx.physics.box2d.Shape.Type;
+
+import static com.dragons.game.utilities.Constants.PPM;
+
+public abstract class Object implements IObject{
+    private Vector2 position;
+    private final IModelType type;
+    private final float width, height;
```

Comment:
Strengt tatt er kanskje ikke bredde og høyde beskrivende for alle objekter. En bombe vil kanskje bare ha radius, så det kan være en tanke å ikke anta det her. Idéelt sett vil isStatic og isSensor også bare plasseres i gameObject fordi du øker modifiabilitien ved å bare ha det der fremfor å eksplisitt ta det med i alle klasser. Hvis jeg vil bruke player-klassen i et annet spill senere så må jeg da f. eks. fjerne dette fra både denne klassen og player-klassen jeg lagde. Ikke en streng kritikk dette, mer en sidenote som jeg ikke vil skal rakke ned på det utrolig bra arbeidet her.